src/core/components/version-stamp.jsx

```diff
@@ -1,12 +1,16 @@
 import React from "react"
 import PropTypes from "prop-types"
 
-const VersionStamp = ({ version }) => {
-  return <small><pre className="version"> { version } </pre></small>
-}
+const VersionStamp = ({ version, children }) => (
+  <div className="info__version-stamp">
+    <small><pre className="version"> { version } </pre></small>
+    { children }
```

Comment:
**Question**: where is this new functionality being used?

Reply:
So, this was done in order to address an inconsistency in the rendered html structure between Oas3 and 2.0 defs.
When using oas3 wrapper - the `Ori` component (this one), was being placed on top of of the oas3 version-stamp which is all wrapped in a new span...
2.0 example:

```
<h2 className="title" >
    <span>{ title }</span>
    <small></small>
</h2>
```

3.0 example
```
<h2 className="title" >
     <span>{ title }</span>
     <span>
         <small></small>
         <small></small>
    </span>
</h2>
```

Currently the oas3 wrapper will always require an extra `span` to wrap the `Ori` component, so there is no way to achieve a consistent structure. This change updates the wrapper to render the oas3 stamp as a sibling of the original stamp.

https://github.com/swagger-api/swagger-ui/pull/5395/commits/c71b377a7e8716d46207821275cbc3d15bd0bad0